.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -212,9 +214,15 @@ stages:
                 Invoke-WebRequest $logsUrl -OutFile "results/logs.zip"
 
                 if ($result.testResult -eq "FAILED") {
-                  throw "*** ERROR: Test $($result.testRunId) failed."
+                  throw "*** ERROR: Test result for test run $($result.testRunId) is failed. Test did not match the defined test criteria."
```

Comment:
```suggestion
                  throw "*** ERROR: Test result for run $($result.testRunId) is 'Failed'. Test did not match the defined test criteria."
```

Reply:
Ok for me - but in case you want to align it with the status then it has to be "FAILED".

Reply:
also fine :)
btw, are we sure the missed criteria are the only reason this could end in failed`?